executorlib/standalone/serialize.py

```
@@ -28,22 +28,24 @@
         pass
 
 
-def serialize_funct_h5(fn: callable, *args: Any, **kwargs: Any) -> Tuple[str, dict]:
+def serialize_funct_h5(
+    fn: callable, fn_args: list = [], fn_kwargs: dict = {}
```

Ruff

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)